packages/sdk/src/schema/threads.ts

```diff
@@ -55,3 +48,56 @@ export const SubThreadsComponent = {
   schema: co.feed(RoomyEntity),
   id: "space.roomy.subthreads.v0",
 };
+
+export const PlainTextContentComponent = {
+  schema: co.map({
+    content: z.string(),
+  }),
+  id: "space.roomy.content.plaintext.v0",
+};
+
+export const UserAccessTimesComponent = {
+  // note that Jazz also tracks access times for all CoValues;
+  // this component allows manual override of access times
+  schema: co.map({
+    createdAt: z.date(),
+    updatedAt: z.date(),
+  }),
+  id: "space.roomy.useraccesstimes.v0",
+};
+
+export const HiddenInComponent = {
+  schema: co.map({
+    hiddenIn: co.list(z.string()), // list of thread IDs where message should be hidden
+  }),
+  id: "space.roomy.hiddenin.v0",
+};
+
+export const ReplyToComponent = {
+  // ID of the message that this message is a reply to
```

Comment:
Looks like you already know this is missing a `schema`, along with `AuthorComponent` and `BranchThreadIdComponent` below.